internal/importing/filepath_test.go

```diff
@@ -24,30 +23,39 @@ func TestFilesOfArgs(t *testing.T) {
 		},
 		// files
 		{
-			[]string{"filepath.go"},
-			[]string{"filepath.go"},
+			[]string{"./filepath_fixtures/first.go"},
+			[]string{"./filepath_fixtures/first.go"},
 		},
 		// directories
 		{
-			[]string{"."},
-			[]string{"filepath.go", "import.go"},
+			[]string{"./filepath_fixtures"},
+			[]string{"filepath_fixtures/first.go", "filepath_fixtures/second.go", "filepath_fixtures/third.go"},
```

Comment:
You use inconsistent filenames. Some start with `.`, some start directly with dir name. Is that intentional?